docs/configuration-options/index.md

```diff
@@ -1452,7 +1452,7 @@ Note that it is not necessary for the listed modules themselves to be part of th
 });
 ```
 
-will put all lodash modules into a manual chunk even if you are only using imports of the form `import get from 'lodash/get'`.
+will put all lodash modules into a `lodash` chunk.
```

Comment:
Removed the note about modules not needing to be part of the module graph since this is no longer true. See the `manual-chunk-not-included` test below. 

The PR doesn't affect the behavior outlined in the docs though. I tested on a repro with lodash using `import get from 'lodash/get'` and the modules were properly grouped together into the `lodash` chunk. So I guess the statement wasn't really accurate to begin with.